### ISSUE TITLE
File load stalls at 64% of Game Objects phase

### ISSUE THREAD

**Issue by TheEvilGreebo** [AnthorNet/SC-InteractiveMap#311]
# Issue
Recurring but somewhat intermittent problem with loading save files.  File loads and then is parsed to 64% and then stalls.  Here are links to two such files that stall.  The problem started after building the steel processing in the north of the dune desert.

The issue seems to be off and on.  It was happening yesterday, then I went to report it, tested one more time and the file loaded, but now it's happening again.

https://drive.google.com/drive/folders/1B6WUc5n8lBHOP5B3JwOCSp6Uv4MAr8Hq?usp=sharing

This occurs even after forced cache refreshing with CTRL-F5 and with clearing cookies/cache using settings.  Browser is Chrome, latest version.

## Replication
Open map viewer, open one of the files in the attached link.

### Actual Result
Process loads to the "Rendering Object" phase in the mid 60s.  Right now its 65 but its been 63 and 64 as well.  It simply freezes and stops.

### Expected Result
For the map to load, of course :)

**Comment by Phrankieboy** [comment]
Having the same issue.  Save load stalls on 'Rendering Objects' at the same place each time.  In my case it's at 35%.

I'm on PC (Epic), U8 Exp.  This problem also happens with earlier saves on my playthrought that had no issues loading before.  This started for me on 27 October.  I was able to load a save first thing in the morning, but in the evening, the problem started.

Not intermittent for me, happens every time.

I've tried clearing cache, checked on both Chrome and Firefox, problem persists.
